uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireColumnPicker.java

```diff
@@ -145,12 +145,7 @@ public void addColumn(ColumnMeta<T> columnMeta) {
     }
 
     protected void sortAndAddColumns(List<ColumnMeta<T>> columnMetas) {
-        // Sort based on preferences applied
-        Collections.sort(columnMetas);
-        //Add the columns based on the preferences
-        for (ColumnMeta meta : columnMetas) {
-            addColumn(meta);
-        }
+        columnMetas.stream().sorted().forEach(meta -> addColumn(meta));
```

Comment:
@cristianonicolai how this fix an ConcurrentModificationException?

Reply:
@ederign the same collection is modified inside the scope of _addColumn_, that's why using streams completely avoids that due to its immutable nature.